react-native/react/native/remote-manager.desktop.js

```diff
@@ -129,28 +142,64 @@ class RemoteManager extends Component {
     })
   }
 
+  showUpdatePromptComponents () {
```

Comment:
same here, the component should expose some remoteComponent factor and this thing shouldn't know this stuff